src/interfaces/special-resolution-interfaces/person-interface.ts

```diff
@@ -0,0 +1,12 @@
+export interface PersonIF {
```

Comment:
Please rename this (and EmptyPerson below) to distinguish them from OrgPersonIF and ApiPersonIF.